CMakeLists.txt

```diff
@@ -58,17 +62,22 @@ set(CMAKE_CXX_EXTENSIONS ON)
 set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 
 include(
-  "${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/git_commit_information.cmake")
+    "${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/git_commit_information.cmake"
+    
+)
 
 # Jenkins integration section dont modify this section if you dont know details
 # about integration with Jenkins!!!
 set(HMI "web" CACHE STRING "HMI type")
 option(
-  EXTENDED_MEDIA_MODE
-  "Turn on and off extended Madia Manager features relates to PulseAudio A2DP and GStreamer"
-  OFF)
+    EXTENDED_MEDIA_MODE
```

Comment:
@vilyukh bring to the form as below:
```
 option(EXTENDED_MEDIA_MODE
      "Turn on and off extended Madia Manager features relates to PulseAudio A2DP and GStreamer" 
      OFF
 )

Reply:
@BSolonenko You can correct it, should run cmake-format with follow argument: --dangle-parens 1 or add in config-file as dangle_parens: True

Reply:
@MaximRybak please make this default